ssh/src/main/scala/gridscale/ssh/package.scala

```diff
@@ -38,16 +36,14 @@ package object ssh {
   }
 
   object SSH {
-    def apply() = Effect(new SSH)
 
-    def apply[T](f: Effect[SSH] ⇒ T) = {
-      val intp = Effect(new SSH())
+    def apply(connectionCache: ConnectionCache) = Effect(new SSH(connectionCache))
```

Comment:
I think a default value should be provided for the connection cache.

Reply:
Agree. I've tried an Option[] = None for the other one, is that fine?